src/plugins/curlget/curlget.c

```diff
@@ -490,6 +490,7 @@ static FILE * fetchFile (Data * data, int fd)
 		curl_easy_cleanup (curl);
 		return NULL;
 	}
+// todo replace VERBOSE
 #if VERBOSE
 	curl_easy_setopt (curl, CURLOPT_VERBOSE, 1L);
 #endif
```

Comment:
Simply remove the whole feature, I don't think this is needed. (If it would be needed, a plugin configuration would be the better choice as it would not require recompiliation.)

```suggestion
```